roxie/udplib/udptrr.cpp

```diff
@@ -1362,6 +1364,11 @@ class CReceiveManager : implements IReceiveManager, public CInterface
                         if (now-lastOOOReport > udpStatsReportInterval)
                         {
                             lastOOOReport = now;
+                            if (unwantedDiscarded > lastUnwantedDiscarded)
```

Comment:
trivial: This doesn't cope with overflow, but if that happens something has gone seriously wrong.